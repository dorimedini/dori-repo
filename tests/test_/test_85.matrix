{
    int a;
    int b=1;

    a = 5+b;
    print a;

    b = 5-a;
    print b;

    matrix m1(2,2) = [1,2;3,4];
    matrix m2(2,2) = [2,3;4,5];
    matrix m3(2,2) = m1 - m2;
    print m3;

    b = 2*2;
    print b;
    b = a*a;
    print b;
    b = 2*a;
    print b;
    b = a*2;
    print b;

    m3 = m1*4;
    print m3;

    m3 = 4*m1;
    print m3;

    a=5;
    b = 5/a;
    print b;

    b = a/5;
    print b;

    b = -a;
    print b;

    m3 = -m3;
    print m3;

}
    
