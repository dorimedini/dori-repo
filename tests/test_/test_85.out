6
-1
[-1,-1;-1,-1]
4
36
12
12
[4,8;12,16]
[4,8;12,16]
1
1
-5
[-4,-8;-12,-16]
